packages/trader/src/Modules/Contract/Containers/contract-replay.tsx

```diff
@@ -0,0 +1,384 @@
+                endEpoch={end_epoch}
+                margin={margin || null}
+                isMobile={isMobile()}
+                enabledNavigationWidget={isDesktop()}
+                enabledChartFooter={false}
+                granularity={granularity}
+                requestAPI={wsSendRequest}
+                requestForget={wsForget}
+                requestForgetStream={wsForgetStream}
+                crosshair={isMobile() ? 0 : undefined}
+                maxTick={isMobile() ? 8 : undefined}
+                requestSubscribe={wsSubscribe}
+                settings={settings}
+                startEpoch={start_epoch}
+                scrollToEpoch={scroll_to_epoch}
+                stateChangeListener={chartStateChange}
+                symbol={symbol}
+                allTicks={all_ticks}
+                topWidgets={ChartTopWidgets}
+                isConnectionOpened={is_socket_opened}
+                isStaticChart={
+                    // forcing chart reload when start_epoch changes to an earlier epoch for ACCU closed contract:
+                    is_accumulator_contract && end_epoch && Number(start_epoch) < Number(prev_start_epoch)
+                }
+                shouldFetchTradingTimes={false}
+                should_zoom_out_on_yaxis={is_accumulator_contract}
+                yAxisMargin={getChartYAxisMargin()}
+                anchorChartToLeft={isMobile()}
+                shouldFetchTickHistory={
+                    getDurationUnitText(getDurationPeriod(contract_info)) !== 'seconds' ||
+                    contract_info.status === 'open'
+                }
+                shouldDrawTicksFromContractInfo={is_accumulator_contract}
+                contractInfo={contract_info}
+                contracts_array={getContractsArray()}
+                isLive={!has_ended}
+                startWithDataFitMode={true}
+            >
+                {is_beta_chart &&
+                    markers_array.map(({ content_config, marker_config, react_key }) => (
+                        <ChartMarkerBeta
+                            key={react_key}
+                            marker_config={marker_config}
+                            marker_content_props={content_config}
+                            is_bottom_widget_visible={isBottomWidgetVisible()}
+                        />
+                    ))}
+                {!is_beta_chart &&
+                    markers_array.map(({ content_config, marker_config, react_key }) => (
+                        <ChartMarker
+                            key={react_key}
+                            marker_config={marker_config}
+                            marker_content_props={content_config}
+                            is_bottom_widget_visible={isBottomWidgetVisible()}
+                        />
+                    ))}
+                {!is_beta_chart && is_accumulator_contract && !!markers_array && (
+                    // @ts-expect-error needs fix
```

Comment:
The problem is that after releasing modificated charts, `accu_barriers_marker_component` might be `undefined` (line 255) and for ` <DelayedAccuBarriersMarker />` it is NOT optional. Currently not know how to fix it better, so add a temporary `@ts-expect-error`.
Please, share your thoughts.